cmake/px4_add_board.cmake

```diff
@@ -261,7 +261,7 @@ function(px4_add_board)
 		foreach(driver ${DF_DRIVERS})
 			list(APPEND config_df_driver_list ${driver})
 
-			if(EXISTS "${PX4_SOUCE_DIR}/platforms/posix/drivers/df_${driver}_wrapper")
+			if(EXISTS "${PX4_SOURCE_DIR}/src/platforms/posix/drivers/df_${driver}_wrapper")
```

Comment:
I'm confused that this wasn't killing the build.

Reply:
CMake probably returns empty string for undefined variable resulting in path which always doesn't exist, which causes df wrapper modules not to be included.

Reply:
Error like this should be detected once we have CI running but that is out of scope for this PR.